src/test/java/com/amadeus/jenkins/plugins/workflow/libs/HttpRetrieverIntegrationTests.java

```diff
@@ -261,12 +271,16 @@ public void retrievesWithPreemptiveAuthIfNeeded() throws Exception {
         p.setDefinition(new CpsFlowDefinition(importScript, true));
 
         WorkflowRun run = j.buildAndAssertSuccess(p);
-        FilePath target = new FilePath(run.getRootDir().toPath().resolve("libs").resolve(libraryName).toFile());
-
-        Assert.assertTrue(target.child("version.txt").exists());
-        Assert.assertTrue(target.child("src").exists());
-        Assert.assertTrue(target.child("vars").exists());
-        Assert.assertTrue(target.child("resources").exists());
+        List<Path> children = Files.list(run.getRootDir().toPath().resolve("libs"))
```

Comment:
this creates a stream holding a file handle - and it is never closed, this is likely a cause of the issue on windows where the clean-up after the test can not happen correctly.

Terminal operators do not close underlying streams.